src/Footer.jsx

```diff
@@ -183,10 +184,17 @@ export const Footer = ({
 				<Bounds>
 					<div className="padding--all">
 						<InlineBlockList
-							className="text--small align--center atMedium_align--left"
+							className="text--small align--center atMedium_align--left footerList-legal"
 							separator="·"
 							items={[
 								<span>{`© Meetup ${new Date().getFullYear()}`}</span>,
+								(legalCopy && <span>{legalCopy}</span>),
+							].filter(item => item)}
```

Comment:
I _think_ I need this, so we don't have an unwanted `<li/>` if a consumer app doesn't pass `legalCopy` in, but will double-check.